httpserver/handlers.go

```diff
@@ -33,7 +33,7 @@ import (
 const apiVersion = "externaldata.gatekeeper.sh/v1alpha1"
 
 func (server *Server) verify(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
-	logrus.Infof("start request %v %v", r.Method, r.URL)
+	logrus.Infof("start request %v %s", r.Method, utils.SanitizeURL(*r.URL))
```

Comment:
Typically this method should be executed only if infof is invoked. I don't know if checking the status before invoking these would be useful. It does make the code more verbose but avoids unnecessary CPU cycles sanitizing URLs if this was never scheduled.

Reply:
I updated the code to store a local variable with the sanitized URL/Method if there is a `Infof` that is guaranteed to run. This should avoid extra method calls for the same input. Is this what you were thinking?